traffic/algorithms/navigation.py

```diff
@@ -267,6 +267,8 @@ def on_runway(self, airport: Union[str, "Airport"]) -> Optional["Flight"]:
     def aligned_on_ils(
         self,
         airport: Union[None, str, "Airport"],
+        max_b_diff: float = 0.1,
```

Comment:
I think we should find a better name if we expose that variable name 😓

Reply:
Je suis preneur de suggestion :)

Reply:
`angle_tolerance` ?